prep-javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,39 @@
+var number1 = 2;
+var number2 = 4;
```

Comment:
If you remember in the Q&A for `prep-javascript-variables`, I mentioned that `var` is an outdated way of creating variables. It still works, but here at LearningFuze, we will only use `const` or `let`. In this specific example these should be `const`